example/app/data/DemoDataSource/plugins/view_selector/roles_school_example/school.entity.json

```diff
@@ -1,4 +1,5 @@
 {
+  "_id": "hogwars-school-meta",
```

Comment:
typo: hogwarts (with t)

also, perhaps rename to `hogwarts-school-district`, since it's not a meta type?

Reply:
if it's used in a test where it tests _meta_ then perhaps it's okay.